src/app/map/page.jsx

```diff
@@ -1,13 +1,18 @@
 "use client";
 
+import dynamic from "next/dynamic";
 import { useEffect, useState } from "react";
-import { Map } from "@/components/Map/Map";
 
-// Remove SSR for MapComponent, Leaflet uses window requiring client-side rendering
+const Map = dynamic(() => import("@/components/Map/Map"), {
```

Comment:
What's thebenfit of using dynamic loading vs static here?

Reply:
https://nextjs.org/docs/pages/building-your-application/optimizing/lazy-loading#with-no-ssr

Reply:
react-leaflet requires the client side "window" to be present, we need to disable server side rendering for nextjs, because on occasions the window object is not present if next decides to SSR.